src/main/cpp/bindings.cpp

```diff
@@ -2677,7 +2681,13 @@ JNIEXPORT jobject JNICALL Java_org_kortforsyningen_proj_SharedPointer_normalizeF
         BaseObjectPtr ptr = cop.as_nullable();
         return specific_subclass(env, operation, ptr, org_kortforsyningen_proj_Type_COORDINATE_OPERATION);
     } catch (const std::exception &e) {
-        rethrow_as_java_exception(env, JPJ_ILLEGAL_ARGUMENT_EXCEPTION, e);
+        try {
+            CRSNNPtr crs = get_shared_object<CRS>(env, operation);
+            BaseObjectPtr ptr = crs.as_nullable();
+            return specific_subclass(env, operation, ptr, org_kortforsyningen_proj_Type_COORDINATE_REFERENCE_SYSTEM);
```

Comment:
@azahnen this code segment does nothing to the CRS. Should it be always converting the CRS to CRS:84?

Reply:
The line that does the actual axis order switch is missing:
```suggestion
            CRSNNPtr crs = get_shared_object<CRS>(env, operation);
            crs = crs->normalizeForVisualization();
            BaseObjectPtr ptr = crs.as_nullable();
            return specific_subclass(env, operation, ptr, org_kortforsyningen_proj_Type_COORDINATE_REFERENCE_SYSTEM);
```